java/libs.nbjavacapi/nbproject/project.properties

```diff
@@ -18,8 +18,8 @@
 javac.source=1.8
 javac.compilerargs=-Xlint -Xlint:-serial
 license.file.override=${nb_all}/nbbuild/licenses/GPL-2-CP
-release.external/nb-javac-jdk-22+33-api.jar=modules/ext/nb-javac-jdk-22-33-api.jar
-release.external/nb-javac-jdk-22+33.jar=modules/ext/nb-javac-jdk-22-33.jar
+release.external/nb-javac-jdk-22.0.1+8-api.jar=modules/ext/nb-javac-jdk-22.0.1_8-api.jar
+release.external/nb-javac-jdk-22.0.1+8.jar=modules/ext/nb-javac-jdk-22.0.1_8.jar
```

Comment:
note for future: JarClassLoader doesn't like the `+` likely due to url path encoding.